src/lucky_record/validations.cr

```diff
@@ -79,7 +79,13 @@ module LuckyRecord::Validations
   # if because there is no T (model class).
   macro included
     private def build_validation_query(column_name, value) : T::BaseQuery
-      T::BaseQuery.new.where(column_name, value)
+      query = T::BaseQuery.new.where(column_name, value)
+      record.try do |r|
+        r.id.try do |id|
+          query = query.where("#{T::TABLE_NAME}.id != ?", id)
```

Comment:
I believe you can do `query.id.not(id)`

Reply:
Yeah let's remove the `private` to get this working 👍. I think that should be fine.

Thanks for digging in and figuring that out!

Reply:
OK. I just pushed another update.